delta/sdk/src/test/scala/ch/epfl/bluebrain/nexus/delta/sdk/projects/ProjectsImplSpec.scala

```diff
@@ -265,5 +265,27 @@ class ProjectsImplSpec extends CatsEffectSpec with DoobieScalaTestFixture with C
 
       results shouldEqual SearchResults(1L, Vector(anotherProjResource))
     }
+
+    "execute the successful init step even if there is a failing init step" in {
+      val successfulScopeInit = ScopeInitializationLog().accepted
+      val failingScopeInit    = new FailingScopeInitializationLog()
+
+      val org     = Label.unsafe(genString())
+      val project = ProjectRef(org, Label.unsafe(genString()))
+
+      val fetchOrg: FetchOrganization = {
+        case `org` => IO.pure(Organization(org, orgUuid, None))
+        case other => IO.raiseError(WrappedOrganizationRejection(OrganizationNotFound(other)))
+      }
+
+      val inits    = Set(failingScopeInit, successfulScopeInit)
+      val projects = ProjectsImpl(fetchOrg, validateDeletion, inits, defaultApiMappings, config, xas, clock)
+
+      val createProject                    = projects.create(project, payload)
+      val assertSuccessfulInitStepExecuted =
+        successfulScopeInit.createdProjects.get.map(p => p shouldEqual Set(project))
+
+      createProject >> assertSuccessfulInitStepExecuted
```

Comment:
Should it assert that a log was recorded?

Reply:
Yes there should be a check for that. The reason I did not add it is because I didn't want to have to deal with the errorStore directly. In my mind the next step is the ability to check the health/errors for a given project which can then be used to do that check

Reply:
I think it's legit to deal with the error store (mocking it probably makes the most sense for this test), but your suggested next step could work too